### ISSUE TITLE
Posttraumatic stress symptom

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6747]
**Preferred term label:**
Posttraumatic stress symptom

**Synonyms**
Posttraumatic stress disorder symptom
PTSD symptom
related 
Posttraumatic stress disorder

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4663500/

**Definition (free text, please give PubMed ID)**
A behavioral or psychological symptom that typically occurs following exposure to one or more traumatic events. Posttraumatic stress disorder (PTSD) symptoms include intrusive recollections (re-experiencing the trauma in flashbacks, memories or nightmares); avoidant and numbing symptoms (including diminished emotions and avoidance of situations that are reminders of the traumatic event); and hyperarousal (including increased irritability, exaggerated startle reactions or difficulty sleeping or concentrating) [3]. 
PMID:26611143

Comment: This term is intended to be used if no additional information is available. If possible, more specific terms should be used.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
